docs/guide/using-doctrine-orm-filters.php

```diff
@@ -239,31 +221,38 @@ final class BookFixtures extends Fixture
     {
         public function load(ObjectManager $manager): void
         {
-            $johnDoe = AnonymousFactory::new(User::class)::createOne(['username' => 'john.doe']);
-            $janeDoe = AnonymousFactory::new(User::class)::createOne(['username' => 'jane.doe']);
-            AnonymousFactory::new(Book::class)::createMany(10, ['user' => $johnDoe]);
-            AnonymousFactory::new(Book::class)::createMany(10, ['user' => $janeDoe]);
+            $userFactory = AnonymousFactory::new(User::class);
+            $johnDoe = $userFactory->create(['username' => 'john.doe']);
+            $janeDoe = $userFactory->create(['username' => 'jane.doe']);
+
+            $bookFactory = AnonymousFactory::new(Book::class);
+            $bookFactory->many(10)->create([
+                'title' => 'title',
```

Comment:
Why such a fake name? Can't it be auto-faked by AnonymousFactory?

Reply:
I thought of using a variable since the factory is used more than once in this function ?